src/views/GroupSettingsView.java

```diff
@@ -281,6 +297,9 @@ public void actionPerformed(ActionEvent e) {
         } else if ("Add Recommended Event".equals(command)) {
             JOptionPane.showMessageDialog(this, "NOT IMPLEMENTED", "Warning", JOptionPane.WARNING_MESSAGE);
             // TODO: Implement Add Recommended Event logic
+        } else if ("EXPORT CALENDAR".equals(command)) {
```

Comment:
Samy is currently working on this usecase, so expect some merge conflicts to occur here, just as a heads up. His PR can be viewed here: #78